pkgs/build-support/rust/default.nix

```diff
@@ -51,11 +51,15 @@
 # case for `rustfmt`/etc from the `rust-sources).
 # Otherwise, everything from the tarball would've been built/tested.
 , buildAndTestSubdir ? null
+
+, verifyCargoDeps ? ""
 , ... } @ args:
 
 assert cargoVendorDir == null -> !(cargoSha256 == "" && cargoHash == "");
 assert buildType == "release" || buildType == "debug";
 
+assert verifyCargoDeps != "" -> throw "`verifyCargoDeps` is not needed as it is the default";
```

Comment:
Does this still allow to set it to false?

Reply:
No? Not sure if I'm misunderstanding what you're asking but `verifyCargoDeps` is a no op anyway?